app/models/AppDatabase.scala

```diff
@@ -1,26 +1,46 @@
 package models
 
+import javax.inject.{Inject, Singleton}
+
 import com.outworkers.phantom.builder.query.CreateQuery
-import com.outworkers.phantom.connectors.{ ContactPoint, CassandraConnection }
 import com.outworkers.phantom.dsl._
-import scala.concurrent.duration._
 
-object Defaults {
-  val connector: CassandraConnection = ContactPoint.local.keySpace("outworkers")
-}
+import scala.concurrent.{Await, Future}
+import scala.concurrent.duration._
 
-class AppDatabase(val keyspace: CassandraConnection) extends Database[AppDatabase](keyspace) {
+@Singleton
+class AppDatabase @Inject()(env: play.api.Environment, override val connector: CassandraConnection) extends Database[AppDatabase](connector) {
 
-  object users extends Users with Connector {
-    override def autocreate(space: KeySpace): CreateQuery.Default[Users, User] = {
+  object beers extends Beers with Connector {
+    override def autocreate(space: KeySpace): CreateQuery.Default[Beers, Beer] = {
       create.ifNotExists()(space)
         .option(default_time_to_live eqs 10)
         .and(gc_grace_seconds eqs 10.seconds)
         .and(read_repair_chance eqs 0.2)
     }
-  }
-  object beers extends Beers with Connector
-}
 
+    def save(beer: Beer): Future[ResultSet] = {
+      beers.insert
+        .value(_.company, beer.company)
+        .value(_.style, beer.style)
+        .value(_.name, beer.name)
+        .future()
+    }
+
+    def initialize(): Unit = {
+      Await.result(beers.autocreate(space).future(), 5000 millis)
```

Comment:
All these methods return future, unless you wait for them there's no guarantee they will be around for anyone trying to read.

We could use:

```scala
val beerList = List(
  Beer(
    company = "budhizer",
    style = "white",
    name = "Summer Bud"
  ),
  ..
)
// and then

for {
  init <- beers.create()
  records <- beers.storeRecords(beerList)
} yield records
```

Reply:
Done